src/logic/addressBook/utils/index.js

```diff
@@ -19,3 +21,15 @@ export const saveAddressBook = async (addressBook: AddressBook) => {
 }
 
 export const getAddressesListFromAdbk = (addressBook: AddressBook) => Array.from(addressBook).map((entry) => entry.address)
+
+export const getNameFromAddressBook = (userAddress: string): string | null => {
```

Comment:
This is a selector, not an util function

Reply:
Solved